app/config/config.yml

```diff
@@ -224,15 +224,15 @@ old_sound_rabbit_mq:
             heartbeat: 5
 
     producers:
-        mailjet_campaign:
+        mailer_campaign:
```

Comment:
We need to create the new exchanges and queues before deploying this. @atamano

Reply:
Same for the worker `-w mailjet_transactional` -> `-w mailer_transactional `

Reply:
and `-w mailjet_campaign` -> `-w mailer_campaign`

Reply:
yes Im on it